webapp-django/crashstats/settings/base.py

```diff
@@ -177,18 +176,33 @@ def path(*dirs):
 # Always generate a CSRF token for anonymous users.
 ANON_ALWAYS = True
 
-LOG_LEVEL = logging.INFO
+LOGGING_LEVEL = config('LOGGING_LEVEL', 'INFO')
 
-HAS_SYSLOG = True  # XXX needed??
-
-LOGGING_CONFIG = None
-
-# This disables all mail_admins on all django.request errors.
-# We can do this because we use Sentry now instead
 LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'handlers': {
+        'console': {
+            'level': LOGGING_LEVEL,
+            'class': 'logging.StreamHandler',
+        },
+    },
     'loggers': {
+        'django': {
+            'handlers': ['console'],
+            'level': LOGGING_LEVEL,
+        },
+        # This disables all mail_admins on all django.request errors because we
+        # use Sentry for that
         'django.request': {
-            'handlers': []
+            'handlers': ['console'],
+        },
+        'py.warnings': {
+            'handlers': ['console'],
```

Comment:
I claim we want to see these and not ignore them. It prints things that we should deal with under the auspices of a future bug. For example:

```
/usr/local/lib/python2.7/site-packages/ratelimit/middleware.py:2: RemovedInDjango19Warning: django.utils.importlib will be removed in Django 1.9.
  from django.utils.importlib import import_module

/app/webapp-django/crashstats/crashstats/urls.py:108: RemovedInDjango19Warning: Default value of 'RedirectView.permanent' will change from True to False in Django 1.9. Set an explicit value to silence this warning.
  pattern_name='monitoring:index'
```

Should we keep this section in?